macro/CreateBCPattern.C

```diff
@@ -1,6 +1,7 @@
 #if !defined(__CLING__) || defined(__ROOTCLING__)
 #include "CommonDataFormat/BunchFilling.h"
 #include "CommonConstants/LHCConstants.h"
+#include "PWGUD/Core/UDFSParser.h"
```

Comment:
Where does this come from? It does not seem to be within O2. I would suggest to move this macro to some other place (O2DPG/utils) if it depends on more than just O2.

Reply:
Yes indeed this depends on O2Physics. So maybe better to put it there somewhere. On the other this PR is an extension of the existing macro CreateBCPattern.C and I think it could be of general interest.
I see three options: 1. make a copy of the modified  CreateBCPattern.C in O2Physics and drop the changes in O2. 2. Move PWGUD/Core/UDFSParser.h to O2. UDFSParser.h allows to parse filling scheme files *.csv. 3. keep the modified CreateBCPattern.C privat.